rinja_parser/src/lib.rs

```diff
@@ -826,6 +826,14 @@ impl<'a> SyntaxBuilder<'a> {
                     "delimiters may not contain white spaces. \
                         The {k} delimiter ({s:?}) contains white spaces",
                 ));
+            } else if is_closing
+                && ['(', ')', '.', '>', '<', '&', '|', '!'].contains(&s.chars().next().unwrap())
```

Comment:
`-`, `+`, `~`, too, because they are used for whitespace control. I think that's the problem in [testing/tests/ui/broken-config.stderr](https://github.com/rinja-rs/rinja/pull/180/files#diff-12dfffd906f3d77d1c4e493b55f70d28ffd45498e2d36bf824e3540fcfac199b).